sentry/api/sentry.api

```diff
@@ -661,10 +661,12 @@ public abstract interface class io/sentry/IScope {
 	public abstract fun endSession ()Lio/sentry/Session;
 	public abstract fun getAttachments ()Ljava/util/List;
 	public abstract fun getBreadcrumbs ()Ljava/util/Queue;
+	public abstract fun getClient ()Lio/sentry/ISentryClient;
```

Comment:
Client now lives on `IScope` and we'll set the default one on global scope in the future (follow up PR).

Reply:
Client is now bound to global scope, as of https://github.com/getsentry/sentry-java/pull/3344 but the code region is changed in later PRs a couple times.

Reply:
Should still be true. The `SentryClient` is set on global scope on `Sentry.init`. Before that there's a `NoOpSentryClient` on global scope. By default there should also only ever be `NoOpSentryClient` on isolation and current scope. A user can however bind any client on any of the scopes.